Slim/Player/Playlist.pm

```diff
@@ -687,6 +687,9 @@ sub stopAndClear {
 }
 
 sub fischer_yates_shuffle {
+
+	srand(time()); # make it really random
```

Comment:
We already have a call to `srand()` in `slimserver.pl`. Could you please compare results if you added the seed there instead of repeatedly calling `srand()` here?

Reply:
Any results?